docs/howto.md

```diff
@@ -211,44 +234,49 @@ It is a good question! Since we are using binary protocol and have serialization
 OZO_PG_BIND_TYPE(std::string, "text")
 ```
 
-`OZO_PG_BIND_TYPE(CPP_TYPE, PG_TYPE)` defines a C++ to built-in PostgreSQL type mapping. It's arguments are:
+`OZO_PG_BIND_TYPE(CPP_TYPE, PG_TYPE)` defines a C++ to built-in PostgreSQL type mapping. Its arguments are:
 
 * `CPP_TYPE` - C++ type.
-* `PG_TYPE` - PostgreSQL type name.
-
-It defines C++ to built-in PostgreSQL type mapping with it's array. The current array represantation in OZO are std::vector, std::array, std::list, so for `std::string` type it could be `std::vector<std::string>`, `std::array<std::string, N>`, `std::list<std::string>`.
+* `PG_TYPE` - PostgreSQL type name (as a raw string literal).
 
-Since the mapping is many **C++** types to one **PostgreSQL** type, you can always map another C++ type as PostgreSQL type.
+It defines a type mapping from the C++ type to the specified built-in PostgreSQL type. This mapping also adds compatibility for the various array representations supported by OZO. The current array represantations in OZO are std::vector, std::array, std::list, so for a type that you bind to `std::string`, you can use `std::vector<std::string>`, `std::array<std::string, N>`, `std::list<std::string>` in your queries.
 
-E.g. we have an in-library `text` type definition like:
+This mapping is many-to-one, allowing you to map multiple C++ types to the same PostgreSQL type. For example, OZO has the built-in mapping for the `text` type like such:
```

Comment:
Minor: You can omit all 'like such' and 'like so' expressions in this document that located before code blocks.
Use the colon sign instead.